python/grapher.go

```diff
@@ -13,6 +14,7 @@ import (
 
 	"sourcegraph.com/sourcegraph/srclib/graph"
 	"sourcegraph.com/sourcegraph/srclib/unit"
+	"sourcegraph.com/sourcegraph/toolchain"
```

Comment:
- [ ] should be "sourcegraph.com/sourcegraph/srclib/toolchain"

If you download goimports, it'll get the imports automatically.